tools/ci_connector_ops/setup.py

```diff
@@ -19,7 +19,8 @@
     python_requires=">=3.9",
     entry_points={
         "console_scripts": [
-            "check-test-strictness-level = ci_connector_ops.check_test_strictness_level:main",
+            "check-test-strictness-level = ci_connector_ops.sat_config_checks:check_test_strictness_level",
+            "requires-connector-team-review = ci_connector_ops.sat_config_checks:requires_connector_team_review"
```

Comment:
Even if it has been reviewed by a team member, won't it still return 1 and make the build not green? 

I'm not sure exactly how it would look, but maybe in the github action we can do something like (pseudocode in python-ish that has to be bash)
```
if script returns 1: 
  if not any([reviewers in connectors_team for reviewers in approving_reviewers])
    fail job
```

Reply:
Yes the outer logic of when to call this command should be built in a GitHub action... 
@pedroslopez was suggesting to at least request for review to our team if the script returns 1. Getting the list of approving reviewers looks a bit trickier. Is is something you've done in GHA already?

Reply:
You're right, I assumed there'd be some sort of readily available `github.event.pull.request.reviewers` variable, but there doesn't seem to be anything regarding reviewers [here](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows). 

Another option that may be easier would be instead of checking the review itself, the action could be the one that requests the team for review if the script returns 1. There seem to be a decent amount of actions available that do that:
* https://github.com/marketplace/actions/auto-request-review seems maintained
  * `Auto-assign the default reviewers if no reviewers are matched to your rules` or `Request review only in certain conditions` is probably the relevant behavior here 
  * https://github.com/marketplace/actions/add-pull-request-reviewer is a bit older but seems simple